MMUL-BASH-01/Exercises/Day-4/Exercise.005.solutions

```diff
@@ -0,0 +1,36 @@
+# Exercise 005 - Declare and initialize an array
+
+1) Declare the array $tarantino
```

Comment:
Still repeating the question instead of writing the solutions, and again everything should be in a script.